apps/src/templates/census2017/CensusTeacherBanner.jsx

```diff
@@ -364,14 +380,14 @@ export default class CensusTeacherBanner extends Component {
         <div>
           <div style={styles.header}>
             <h2 style={styles.title}>Add {schoolName} to our map!</h2>
-            <p style={styles.updateSchool}>Not teaching at this school anymore? <a onClick={this.showSchoolInfoForm}>Update here</a></p>
+            <p style={styles.updateSchool}>Not teaching at this school anymore? <a style={styles.updateSchoolLink} onClick={this.showSchoolInfoForm}>Update here</a></p>
```

Comment:
nit: split this line for readability (and to fit [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react)):

```jsx
<p style={styles.updateSchool}>
  Not teaching at this school anymore? 
  <a style={styles.updateSchoolLink} onClick={this.showSchoolInfoForm}>
    Update here
  </a>
</p>
```

Reply:
Done.